api/kyverno/v1alpha2/common_types.go

```diff
@@ -0,0 +1,101 @@
+package v1alpha2
+
+import (
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type CleanupSpec struct {
+	// The schedule in Cron format
+	// +optional
```

Comment:
Still not really optional

Reply:
We will need to validate the schedule expression at some point too

Reply:
Did it,

Yes I have the logic for that, which I added previously, here: https://github.com/NikhilSharmaWe/kyverno/blob/5e5da9e88c76f24ac1ef155c573a979b58da6a38/pkg/policy/validate.go#L276-L281